src/js/components/Layout/Header.jsx

```diff
@@ -38,11 +38,28 @@ class Header extends Component {
   }
 
   render() {
+    let helpScoutBeacon = <div />;
+    if (this.props.isHelpScoutEnabled) {
```

Comment:
Please move it to a separate method. It should be: 
```
getHelpScoutBeacon() {
  if (this.props.isHelpScoutEnabled) {
    return (
       <LiveChatLoaderProvider provider="helpScout" providerKey={this.props.helpScoutKey}>
          <HelpScout
            color={this.props.helpScoutColor}
            horizontalPosition="right"
            icon="question"
            idlePeriod="0"
            zIndex="1050"
          />
        </LiveChatLoaderProvider>
    );
  }
  
  return null;
} 
```
and move it above `render` method and replace `{helpScoutBeacon}` with `{this.helpScoutBeacon()}`